packages/docusaurus/src/client/theme-fallback/Loading/index.js

```diff
@@ -7,38 +7,7 @@
 
 import React from 'react';
 
-export default ({error, retry, pastDelay}) => {
-  if (error) {
-    return (
-      <div
-        style={{
-          align: 'center',
-          color: '#fff',
-          backgroundColor: '#fa383e',
-          borderColor: '#fa383e',
-          borderStyle: 'solid',
-          borderRadius: '0.25rem',
-          borderWidth: '1px',
-          boxSizing: 'border-box',
-          display: 'block',
-          padding: '1rem',
-          flex: '0 0 50%',
-          marginLeft: '25%',
-          marginRight: '25%',
-          marginTop: '5rem',
-          maxWidth: '50%',
-          width: '100%',
-        }}>
-        <p>{error.message}</p>
-        <div>
-          <button type="button" onClick={retry}>
-            Retry
-          </button>
-        </div>
-      </div>
-    );
-  }
-
```

Comment:
Without removing this I got this when seeing the SSR site:
<img width="1674" alt="Screen Shot 2021-03-29 at 13 34 15" src="https://user-images.githubusercontent.com/4290500/112879837-ace3be00-9097-11eb-9492-5a097a9ee981.png">
The error is saying that it cannot require some webpack generated chunks, while those files are clearly in the right folder...
The site seems to run fine, but looks like the static html generation is broken now :(